<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditWindow2</class>
 <widget class="QWidget" name="EditWindow2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>627</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="labelBalance">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>340</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Balance:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelSecondNumber">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Second Name (Optional):</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelRoomNum">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Room #:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelSchool">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>310</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>School:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>410</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCourse">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>310</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Course:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelHomeAddress">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>280</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Home Address:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>711</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>45</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>EDIT PROFILE</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditFirstName">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>160</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditHomeAddress">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>280</y>
     <width>451</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditMiddleName">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>220</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditCourse">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>310</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditBalance">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>340</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditSecondName">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>190</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>141</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Select</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7A</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7B</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7C</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>77</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>99</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="labelLastName">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>250</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Last Name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditLastName">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>250</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelFirstName">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>First Name:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSave">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>410</y>
     <width>111</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelMiddleName">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Middle Name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditSchool">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>310</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxFemale">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>130</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Female</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxMale">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>130</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Male</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelGender">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Gender:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
